KsApiTests/models/ProjectTests.swift

```diff
@@ -23,34 +23,39 @@ final class ProjectTests: XCTestCase {
   func testEndsIn48Hours_WithJustLaunchedProject() {
 
     let justLaunched = Project.template
-      |> Project.lens.dates.launchedAt .~ Date().timeIntervalSince1970
+      |> Project.lens.dates.launchedAt .~ Date(timeIntervalSince1970: 1475361315).timeIntervalSince1970
 
-    XCTAssertEqual(false, justLaunched.endsIn48Hours)
+    XCTAssertEqual(false, justLaunched.endsIn48Hours(today: Date(timeIntervalSince1970: 1475361315)))
   }
 
   func testEndsIn48Hours_WithEndingSoonProject() {
     let endingSoon = Project.template
-      |> Project.lens.dates.deadline .~ (Date().timeIntervalSince1970 - 60.0 * 60.0)
+      |> Project.lens.dates.deadline .~ (Date(timeIntervalSince1970: 1475361315)
+        .timeIntervalSince1970 - 60.0 * 60.0)
 
-    XCTAssertEqual(true, endingSoon.endsIn48Hours)
+    XCTAssertEqual(true, endingSoon.endsIn48Hours(today: Date(timeIntervalSince1970: 1475361315)))
   }
 
   func testEndsIn48Hours_WithTimeZoneEdgeCaseProject() {
     let edgeCase = Project.template
-      |> Project.lens.dates.deadline .~ (Date().timeIntervalSince1970 - 60.0 * 60.0 * 47.0)
+      |> Project.lens.dates.deadline .~ (Date(timeIntervalSince1970: 1475361315)
+        .timeIntervalSince1970 - 60.0 * 60.0 * 47.0)
 
-    XCTAssertEqual(true, edgeCase.endsIn48Hours)
+    XCTAssertEqual(true, edgeCase.endsIn48Hours(today: Date(timeIntervalSince1970: 1475361315)))
   }
 
   func testIsPotdToday_OnPotd() {
-    let potdAt = Calendar.current.startOfDay(for: Date()).timeIntervalSince1970
+    let potdAt = Calendar.current.startOfDay(for:
+      Date(timeIntervalSince1970: 1475361315)).timeIntervalSince1970
     let potd = Project.template
       |> Project.lens.dates.potdAt .~ potdAt
 
-    XCTAssertEqual(true, potd.isPotdToday())
-    XCTAssertEqual(true, potd.isPotdToday(today: Date()))
-    XCTAssertEqual(false, potd.isPotdToday(today: Date(timeIntervalSinceNow: 60.0 * 60.0 * 24)))
-    XCTAssertEqual(false, potd.isPotdToday(today: Date(timeIntervalSinceNow: -60.0 * 60.0 * 24)))
+//    XCTAssertEqual(true, potd.isPotdToday())
```

Comment:
This can be removed because this test will always fail without `today` passed in. Just left here for comment.